app/controllers/CheckYourAnswersController.scala

```diff
@@ -56,10 +56,11 @@ class CheckYourAnswersController @Inject()(checkYourAnswersPage: CheckYourAnswer
           for {
             previousUploads <- uploadJourneyRepository.getUploadsForJourney(request.session.get(SessionKeys.journeyId))
           } yield {
-            val previousUploadsFileName = previousUploads.map( _.map( file => file.uploadDetails.map( details => details.fileName)))
-            val answersFromSession = sessionAnswersHelper.getAllTheContentForCheckYourAnswersPage(Some(previousUploadsFileName.getOrElse(Seq.empty).collect{
+            val previousUploadsFileName = previousUploads.map(_.map(file => file.uploadDetails.map(details => details.fileName)))
+            val x = previousUploadsFileName.getOrElse(Seq.empty).collect {
```

Comment:
This is my mistake :sorry: . But can you make `x` to be something like `fileNames`?